src/cdk-experimental/ui-patterns/accordion/accordion.ts

```diff
@@ -173,7 +179,16 @@ export class AccordionTriggerPattern {
     this.pointerdown().handle(event);
   }
 
-  private _getItem(e: PointerEvent) {
+  /** Handles focus events on the trigger. This ensures the tabbing changes the active index. */
+  onFocus(event: FocusEvent): void {
```

Comment:
Is this different from what is being handled in `pointerdown`? I ask because I'm not sure I understand the case where a focus event is happening (not triggered by us) that isn't a `pointerdown` event